app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor not vaccinated")
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Visitors vaccination is outdated")
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError("Visitor not wearing a mask")
+        return f"Welcome to {self.name}"
```

Comment:
To make the code cleaner, it's better to avoid unnecessary `else` statements. In this case, you can remove the `else` statement and unindent the following line. Here is how you can do it:

```python
if condition:
    return 'We are going to the cinema!'
return 'We are not going to the cinema('
```